cellxgene_schema_cli/tests/fixtures/examples_validate.py

```diff
@@ -316,9 +329,10 @@
         ["spike-in", False, "ERCC-00002 (spike-in control)", "NCBITaxon:32630", 1061, "synthetic"],
         ["gene", False, "MACF1", "NCBITaxon:9606", 2821, "protein_coding"],
         ["gene", False, "Trp53", "NCBITaxon:10090", 1797, "protein_coding"],
-        ["gene", False, "S", "NCBITaxon:2697049", 3822, "protein_coding"],
+        ["gene", False, "S_ENSSASG00005000004", "NCBITaxon:2697049", 3822, "protein_coding"],
+        ["gene", False, "FBtr0472816_df_nrg", "NCBITaxon:7227", 22, ""],
```

Comment:
why is the feature_type an empty string for this one?

Reply:
i started a discussion here to confirm we can generalize the logic a bit: https://czi-sci.slack.com/archives/C02TSE92DBM/p1729886608538879